t/BikeSharingCitiBikeNYC.t

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use DDG::Test::Spice;
+
+ddg_spice_test(
+    [qw( DDG::Spice::BikeSharing::CitiBikeNYC )],
+    'citi bike ny' => test_spice(
+        '/js/spice/bike_sharing/citi_bike_nyc/ny',
+        call_type => 'include',
+        caller => 'DDG::Spice::BikeSharing::CitiBikeNYC',
+    is_cached => 1,
+    ),
```

Comment:
@iambibhas added the test (and modified the code a bit to actually capture the geographic reference).

Still, I'm not doing anything with that in the js... It would be nice to be able to sort/filter. Any ideas?